packages/react/src/components/Storage/FileUploader/Tracker/index.tsx

```diff
@@ -1,18 +1,115 @@
 import React from 'react';
-import { Card, View } from '../../../../primitives';
+import { humanFileSize } from '@aws-amplify/ui';
+import { TrackerProps } from '../types';
+import {
+  Card,
+  Flex,
+  View,
+  Image,
+  Text,
+  Button,
+  TextField,
+} from '../../../../primitives';
+import { CloseIcon, EditIcon, fileIcon } from '../Previewer/PreviewerIcons';
 
 export function Tracker({
-  children,
-}: {
-  children?: React.ReactNode;
-}): JSX.Element {
+  file,
+  hasImage,
+  url,
+  onChange,
+  onCancel,
+  name,
+}: TrackerProps): JSX.Element {
+  const [isEditing, setIsEditing] = React.useState(false);
+  if (!file) return null;
+
+  const { size } = file;
+
+  let icon = (
+    <View className="amplify-fileuploder--img-placeholder">{fileIcon}</View>
```

Comment:
should this be
```suggestion
    <View className="amplify-fileuploder__img-placeholder">{fileIcon}</View>
```

Reply:
Yes, that is my sloppy PoC code. I'm going through and updating all the classnames and styling in separate PRs